### ISSUE TITLE
Corrections for #3436 LPH 6112 Marado and #3436 #3938 K-SAAM

### ISSUE THREAD

**Issue by Ma-ryang** [PygmalionOfCyprus/cmo-db-requests#686]
### DB Selector

DB3K

### Affected DBID(s)

#3436 #3938

### Summary of Changes

In the latest DB version, #3436 LPH 6112 Marado is armed with incorrectly implemented missiles #3869 K-SAAM.
K-SAAM (해궁) is basically a vertical launch shortrange SAM system that will replace the RIM-116 system in the ROKN, 
The maximum range is 20km and IIR and active RF seekers are used, but the current two K-SAAM #3869 and #3938 are implemented with a mix of correct and incorrect data.

#3938 K-SAAM is seems correct with its range, climb rate and target altitude, target speed.
however according to the published information, the length, span, diameter and weight is wrong, especially weight is too light also missing active radar seeker also guidance seems wrong too (missing RF seeker guidance).
#3869 K-SAAM is correct with its length, span, diameter and weight, but the maximum range is wayy too far compare with IRL
(20km vs 200km), Also Interceptable target speed is too high, K-SAAM is capable of intercepting supersonic anti-ship missiles, but intercepting targets at the 6500 knot level is too much, should be 2000 knot same as #3938. also properties is missing 
Capable vs Seaskimmer type, and vaild target is missing surface ship. sensor and guidance system seem correct 

IMO it seems good to combine the correct data that reflect the real life perfomance in both K-SAAM or consolidation of both missile..
In conclusion, K-SAAM should be implemented like this shopped image

![1](https://user-images.githubusercontent.com/99650102/158527241-ab745c18-1d08-424a-8537-e0b8938b457a.png)
![Desktop Screenshot 2022 03 16 - 15 04 25 71](https://user-images.githubusercontent.com/99650102/158527314-5478a762-7b43-47ad-820d-86e3bc69db98.png)
-
![Desktop Screenshot 2022 03 16 - 15 19 00 21](https://user-images.githubusercontent.com/99650102/158531310-fe4927dc-ad05-4576-9f39-c420422ba5bf.png)
![Desktop Screenshot 2022 03 16 - 15 19 09 40](https://user-images.githubusercontent.com/99650102/158531320-7cc28176-e771-41cb-8a97-153ad3d600e0.png)

![Desktop Screenshot 2022 03 16 - 15 53 11 92](https://user-images.githubusercontent.com/99650102/158533379-6ad7945c-6d4c-48d5-850a-5b86234b9b86.png)

### Sources

https://www.add.re.kr/board?menuId=MENU02830&siteId=SITE00003
https://www.add.re.kr/board?menuId=MENU02649&siteId=SITE00002 (K-SAAM catalog pdf DL is only available in KR web)
https://www.lignex1.com/web/eng/product/product.do?category=01&part=02&model=04#categoryMenu
https://www.navalnews.com/naval-news/2019/01/south-korea-set-to-deploy-new-haegung-shipborne-sam-system/
https://www.navalnews.com/naval-news/2021/12/south-koreas-new-k-saam-sam-system-ready-for-mass-production/

**Comment by PygmalionOfCyprus** [comment]
Fixed 494.

Sorry about that. I actually noticed this myself the other day and was working on fixing it independently. Somehow we created a duplicate K-SAAM (3938) that was just...not a good version. I've merged data from the two together as you suggested and deprecated 3938.

One change I've made: rather than use the bitty AIM-9 warhead from the RIMs, I've given it a 90kg Frag warhead like an SM-2, from which it appears to be more closely derived (despite ostensibly replacing the RIM-116s).
